jekyll/_cci2/server/installation/migrate-from-server-3-to-server-4.adoc

```diff
@@ -79,50 +82,56 @@ The instructions below guide you through cloning the repository containing the K
 
 [source,shell]
 ----
-############ HELM Commands ################
-Output file is here - <path>/kots-exporter/output/helm-values.yaml
+############ Upgrade Commands ################
+Before upgrading to 4.0, follow below steps:
 
-To upgrade Postgres Chart(v11.6.0), follow below steps -
+Your helm values file with your installation config is here:
+- <path>/kots-exporter/output/helm-values.yaml
 -------------------------------------------------------------------------
-export POSTGRESQL_PASSWORD=$(kubectl get secret --namespace <namespace> postgresql -o jsonpath="{.data.postgres-password}" | base64 --decode)
+## Progres Chart Upgrade Preparation
+Upgrading to server CircleCI server 4.0 includes upgrading the Posgres chart
+Before upgrading, we need to prepare your postgres instance.
+This is only needed if your Postgres Instance is not externalized.
+
+# Collect your postgres user's password and the PVC attached to your postgres instance
```

Comment:
```suggestion
[source,shell]
----
# Collect your Postgres user's password and the PVC attached to your Postgres instance
```

Reply:
this is part of the output of the migration command